tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java

```diff
@@ -139,6 +140,15 @@ private enum KeyAction {
         TYPED
     }
 
+    @Before
+    public void before() {
+        double x = stage.getX() + stage.getWidth();
+        double y = stage.getY() + stage.getHeight();
+        Util.runAndWait(() -> {
+            robot.mouseMove(x,y);
```

Comment:
Since there is only one line to be executed in this lambda, this can also be written as one line.
-> `Util.runAndWait(() -> robot.mouseMove(x,y));`